packages/braid-design-system/src/lib/components/MenuItemCheckbox/MenuItemCheckbox.screenshots.tsx

```diff
@@ -22,138 +23,274 @@ export const screenshots: ComponentScreenshot = {
     {
       label: 'Default',
       Example: () => (
-        <Box display="flex">
-          <Menu {...defaultProps}>
-            <MenuItemCheckbox checked={false} onChange={() => {}}>
-              Checkbox
-            </MenuItemCheckbox>
-            <MenuItemCheckbox checked={true} onChange={() => {}}>
-              Checkbox
-            </MenuItemCheckbox>
-          </Menu>
-        </Box>
+        <Inline space="medium">
+          <Box display="flex">
+            <Menu {...defaultProps}>
+              <MenuItemCheckbox checked={false} onChange={() => {}}>
+                Checkbox
+              </MenuItemCheckbox>
+              <MenuItemCheckbox checked={true} onChange={() => {}}>
+                Checkbox
+              </MenuItemCheckbox>
+            </Menu>
+          </Box>
+          <Box display="flex">
+            <Menu {...defaultProps} size="small">
+              <MenuItemCheckbox checked={false} onChange={() => {}}>
+                Checkbox
+              </MenuItemCheckbox>
+              <MenuItemCheckbox checked={true} onChange={() => {}}>
+                Checkbox
+              </MenuItemCheckbox>
+            </Menu>
+          </Box>
```

Comment:
Not sure the flex wrappers are needed inside an `Inline` now.
```suggestion
          <Menu {...defaultProps}>
            <MenuItemCheckbox checked={false} onChange={() => {}}>
              Checkbox
            </MenuItemCheckbox>
            <MenuItemCheckbox checked={true} onChange={() => {}}>
              Checkbox
            </MenuItemCheckbox>
          </Menu>
          <Menu {...defaultProps} size="small">
            <MenuItemCheckbox checked={false} onChange={() => {}}>
              Checkbox
            </MenuItemCheckbox>
            <MenuItemCheckbox checked={true} onChange={() => {}}>
              Checkbox
            </MenuItemCheckbox>
          </Menu>
```